src/entities/Suggestion.java

```diff
@@ -0,0 +1,16 @@
+package src.entities;
+
+import java.util.ArrayList;
+
+public class Suggestion {
+    public ArrayList<String> suggestions;
```

Comment:
The type should be List<String>, not ArrayList<String>. Variable types and return types should be interfaces when possible.